src/modules/claim/claim.controller.ts

```diff
@@ -249,27 +248,26 @@ export class ClaimController {
     summary: 'returns claims for Requester with given DID',
   })
   @ApiQuery({
-    name: 'accepted',
+    name: 'isAccepted',
     required: false,
     description:
       '**true** - show only accepted <br> **false** - show only pending',
   })
   @ApiQuery({
-    name: 'namespace',
+    name: 'parentNamespace',
```

Comment:
@JGiter Does the use of `parentNamespace` make sense here? I think that `namespace` is actually more clear. In what sense is it the `parent`?

Reply:
You are right. This is just app or org where corresponding role is defined